kentikapi/client.go

```diff
@@ -150,3 +173,59 @@ func makeRetryingClientConfig(c Config) httputil.ClientConfig {
 		RetryCfg: c.RetryCfg,
 	}
 }
+
+//nolint: gosec
+func makeConnForSynthetics(c Config) grpc.ClientConnInterface {
+	if c.SyntheticsGRPCAPIURL == "" {
+		c.SyntheticsGRPCAPIURL = syntheticsGRPCAPIURL
+		config := &tls.Config{}
+		conn, err := grpc.Dial(
+			syntheticsGRPCAPIURL,
+			grpc.WithTransportCredentials(credentials.NewTLS(config)),
+			grpc.WithUnaryInterceptor(c.createUnaryInterceptor()),
+		)
+		if err != nil {
+			log.Printf("cannot connect to api url for synthetics: %v", err)
+		}
+		return conn
+	}
+	conn, err := grpc.Dial(c.SyntheticsGRPCAPIURL, grpc.WithInsecure())
+	if err != nil {
+		log.Printf("cannot connect to api url for synthetics: %v", err)
+	}
+	return conn
+}
+
+//nolint: gosec
+func makeConnForCloudExport(c Config) grpc.ClientConnInterface {
```

Comment:
Those 2 makeConn methods can be merged into one method with URL input parameter.

Reply:
Done